proto/docker/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM golang:1.13
+
+ARG PROTOC_VERSION=3.15.8
+ARG PROTOC_GEN_DOC_VERSION=1.4.1
+
+RUN apt-get update && \
+    apt-get -y -q install unzip make
+
+RUN curl -OL https://github.com/protocolbuffers/protobuf/releases/download/v${PROTOC_VERSION}/protoc-${PROTOC_VERSION}-linux-x86_64.zip && \
+    unzip -o protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local bin/protoc && \
+    unzip -o protoc-${PROTOC_VERSION}-linux-x86_64.zip -d /usr/local include/* && \
+    rm -rf protoc-${PROTOC_VERSION}-linux-x86_64.zip
+
+RUN go get -u github.com/golang/protobuf/protoc-gen-go && \
+    go get -u github.com/pseudomuto/protoc-gen-doc/cmd/protoc-gen-doc && \
+    go get -u github.com/envoyproxy/protoc-gen-validate
+
+RUN cd ${GOPATH}/src/github.com/envoyproxy/protoc-gen-validate && \
```

Comment:
📝 **[hadolint]** <[SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Double quote to prevent globbing and word splitting.